html-forms/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="#" method="#" id="coding-form">
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <input type="text" name="email" id="email">
+
+      <div>
+        <label for="why">Why is coding cool?</label>
+      </div>
+      <select name="why">
```

Comment:
make sure to add an `id` attribute which matches the `for` attribute on the select input

Reply:
![image](https://user-images.githubusercontent.com/97073517/151487211-c4dc3e20-681a-4b16-afd5-a67e0fdb4318.png)